File: java/Array-1/maxEnd3.java
public int[] maxEnd3(int[] nums) {
  if(nums[0] > nums[2]){
    nums[1] = nums[0];
    nums[2] = nums[0];
  } else {
    nums[0] = nums[2];
    nums[1] = nums[2];
  }
  return nums;
}

File: python/List-1/max_end3.py
def max_end3(nums):
  if(nums[0] > nums[len(nums)-1]):
    for i in range(len(nums)):
      nums[i] = nums[0]
  else:
    for i in range(len(nums)):
      nums[i] = nums[len(nums)-1]
  return nums
